jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/JettyHttpServer.java

```diff
@@ -307,7 +307,7 @@ public void start() {
     @Override
     public void close() {
         try {
-            log.log(Level.INFO, String.format("Shutting down server (graceful=%b)", isGracefulShutdownEnabled()));
+            log.log(Level.INFO, String.format("Shutting down server (graceful=%b, timeout=%.1fs)", isGracefulShutdownEnabled(), server.getStopTimeout()/1000d));
```

Comment:
If the timeout is irrelevant when graceful=false, it should not be printed.

Reply:
The timeout still impacts other parts of Jetty (stopTimeout is generic concept for all `LifeCycle` objects in Jetty).